### PR TITLE
deCLAW Crayfish for  #1267 and #1314

### PR DESCRIPTION
**Pull request by manez**
**GitHub Issue**: https://github.com/Islandora/documentation/issues/1267 and https://github.com/Islandora/documentation/issues/1314

# What does this Pull Request do?

Cleans out the last few traces of CLAW in urls and references. This should get it all!

# What's new?
New mascot for Recast, a few new sections in submodule readmes. A lot less CLAW.

# How should this be tested?

Preview the files and verify that the links still work.

# Interested parties
@jonathangreen new maintainer 😄

### CODE REVIEW

File: Homarus/static/convert.ttl [hunk 1]
```diff
@@ -2,7 +2,7 @@
 @prefix owl:<http://www.w3.org/2002/07/owl#> .
 @prefix ebucore:<http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#> .
 @prefix ldp:<http://www.w3.org/ns/ldp#> .
-@prefix islandora:<http://islandora.ca/CLAW#> .
+@prefix islandora:<http://islandora.ca/islandora8> .
```
Comment (whikloj) [inline Homarus/static/convert.ttl]: Needs a `#` after `islandora8`.

File: Houdini/static/convert.ttl [hunk 2]
```diff
@@ -2,7 +2,7 @@
 @prefix owl:<http://www.w3.org/2002/07/owl#> .
 @prefix ebucore:<http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#> .
 @prefix ldp:<http://www.w3.org/ns/ldp#> .
-@prefix islandora:<http://islandora.ca/CLAW#> .
+@prefix islandora:<http://islandora.ca/islandora8> .
```
Comment (whikloj) [inline Houdini/static/convert.ttl]: Needs a `#` after `islandora8`.

File: Houdini/static/identify.ttl [hunk 3]
```diff
@@ -2,7 +2,7 @@
 @prefix owl:<http://www.w3.org/2002/07/owl#> .
 @prefix ebucore:<http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#> .
 @prefix ldp:<http://www.w3.org/ns/ldp#> .
-@prefix islandora:<http://islandora.ca/CLAW#> .
+@prefix islandora:<http://islandora.ca/islandora8> .
```
Comment (whikloj) [inline Houdini/static/identify.ttl]: Needs a `#` after `islandora8`.

File: Hypercube/static/hypercube.ttl [hunk 4]
```diff
@@ -2,7 +2,7 @@
 @prefix owl:<http://www.w3.org/2002/07/owl#> .
 @prefix ebucore:<http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#> .
 @prefix ldp:<http://www.w3.org/ns/ldp#> .
-@prefix islandora:<http://islandora.ca/CLAW#> .
+@prefix islandora:<http://islandora.ca/islandora8> .
```
Comment (whikloj) [inline Hypercube/static/hypercube.ttl]: Needs a `#` after `islandora8`.

File: README.md [hunk 5]
```diff
@@ -1,10 +1,12 @@
 # ![Crayfish](https://cloud.githubusercontent.com/assets/2371345/15409657/2dfb463a-1dec-11e6-9089-06df94ef3f37.png) Crayfish
 
 [![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%205.6-8892BF.svg?style=flat-square)](https://php.net/)
```
Comment (whikloj) [inline README.md]: Should be (I think)
```
https://img.shields.io/badge/php-%3E%3D%207.2-8892BF.svg?style=flat-square
```

File: Recast/static/recast.ttl [hunk 6]
```diff
@@ -1,6 +1,6 @@
 @prefix apix: <http://fedora.info/definitions/v4/api-extension#> .
 @prefix ldp: <http://www.w3.org/ns/ldp#> .
-@prefix islandora: <http://islandora.ca/CLAW#> .
+@prefix islandora: <http://islandora.ca/islandora8> .
```
Comment (whikloj) [inline Recast/static/recast.ttl]: Needs a `#` after `islandora8`.

**Comment by manez** [comment]
There were similar issues across other .ttl files. I think I got them all.

File: Recast/README.md [hunk 7]
```diff
@@ -1,3 +1,4 @@
+# ![Recast Mascot](https://user-images.githubusercontent.com/2371345/67399869-3ded6600-f583-11e9-8fec-8e82d57a6d4d.png)
```
Comment (whikloj) [inline Recast/README.md]: 👍

**Pull request merged**